_data/taps/versions/ms-teams.yml

```diff
@@ -0,0 +1,22 @@
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/integration-versions/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#     MS TEAMS VERSIONS      #
+# -------------------------- #
+
+latest-version: "1"
+
+released-versions:
+  - number: "1"
+    status: "beta" ## beta, released, deprecated
+    date-released: ""
```

Comment:
Nothing to do here yet - just leaving a note so we don't forget to fill this in when the beta is ready to go